js/deno.ts

```diff
@@ -55,6 +55,13 @@ export { run, RunOptions, Process, ProcessStatus } from "./process";
 export { inspect } from "./console";
 export const args: string[] = [];
 
+// TODO Don't expose Console nor stringifyArgs.
```

Comment:
We may have to do a _test_ build of Deno, like we do for Rust to deal with some of these unit tests in future.

Reply:
I would also mark all of these with `@internal` so they don't leak into the `lib.deno_runtime.d.ts` like the compiler below.

Reply:
Unfortunately that doesn't work:
```
/Users/rld/src/deno/js/net_test.ts:4:10 - error TS2305: Module '"deno"' has no exported member 'deferred'.

4 import { deferred } from "deno";
           ~~~~~~~~
```
However, I have added JSDOC comments that note these will be removed.